### ISSUE TITLE
Status: focused entry should use orange border when using the high contrast theme

### ISSUE THREAD

**Issue by lszomoru** [microsoft/vscode#99350]
To be consistent with the rest of the workbench we should consider:
- Use the orange borders for the active element instead of the blue one
- Remove the background color as I do not see it being used anywhere else for the active element

**Comment by isidorn** [comment]
I agree this makes sense.
We are open for a PR that fixes this.
Code pointer https://github.com/microsoft/vscode/blob/735e08aa9442d73c327811b33383898b2663e1a7/src/vs/workbench/browser/parts/statusbar/statusbarPart.ts#L900

Basically we should I think use a different color there and make sure it is also applied on the whole status bar.
Adding help wanted and if nobody tackles this I can also look into it in the future

**Comment by bpasero** [comment]
How is this different from https://github.com/microsoft/vscode/issues/99247 ?

**Comment by isidorn** [comment]
That one I used blue border for the focus items. Lad thinks it should be orange to make it clearer and consistent.

**Comment by bpasero** [comment]
Yes agree, it should be this orange color we also have in tabs:

![image](https://user-images.githubusercontent.com/900690/83761278-63dcfc00-a676-11ea-91b1-e430b89c4816.png)

**Comment by isidorn** [comment]
Thanks to @jeanp413 this is now fixed
To verify, turn on High Contrast theme and:
1. Make sure we use the orange border in the status bar both for the whole bar and for individual entries
2. Make sure that hover feedback is a yellow doted line
